arch/arm64/core/Kconfig

```diff
@@ -193,6 +193,15 @@ config EXCEPTION_DEBUG
 config XIP
 	select AARCH64_IMAGE_HEADER
 
+config SET_VMPIDR_EL2
+	bool "Set VMPIDR_EL2 at EL2 stage"
+	default n
```

Comment:
`default n` is redundant and should not be specified.

Also this config should be prefixed `ARM_` or `ARM64_` since this is specific to this arch.